config/locales/en/browse.yml

```diff
@@ -4,9 +4,25 @@ en:
     all_categories: All categories
     popular_tasks: Popular tasks
     topics: Topics
-    check_benefits_and_financial_support: Check benefits and financial support you can get
-    hmrc_online_services: "HMRC online services: sign in or set up an account"
-    self_assessment_tax_returns: Self Assessment tax returns
-    pay_employers_paye: "Pay employers' PAYE"
     description: Find the government services, forms and accounts you need to use
     title: Services and information
+    popular_links:
+      benefits:
+        - title: Check benefits and financial support you can get
+          url: /check-benefits-financial-support
+          analytics_label_key: check_benefits_and_financial_support
+      business:
+        - title: "HMRC online services: sign in or set up an account"
+          url: /log-in-register-hmrc-online-services
+          analytics_label_key: hmrc_online_services
+        - title: Self Assessment tax returns
+          url: /self-assessment-tax-returns
+          analytics_label_key: self_assessment_tax_returns
+        - title: "Pay employers' PAYE"
+          url: /pay-paye-tax
+          analytics_label_key: pay_employers_paye
+    analytics_labels:
```

Comment:
I think we could simplify here. The `analytics_label_key` is storing a reference to a field in `analytics_labels` which just stores the link title. But you've already got that string stored in the `title` field. So let's just use the `title` field and remove the duplication.

ie remove all the `analytics_label_key`'s and the `analytics_labels` hash. We only need two keys:`title` and `url`.

Reply:
Removed.